### PR TITLE
Feature/update deps

### PR DESCRIPTION
**Pull request by stephencattaneo**
- [x] update deps to remove axios version with vuln
- [x] fix types for login method

### CODE REVIEW

File: src/sso.js [hunk 1]
```diff
@@ -40,7 +40,6 @@ export function getProviderLink({ provider, redirect, providerId }) {
  * @param {String} [providerId] Provider ID of custom provider (only required for `{ provider: 'custom sso' }`)
  */
 export function signonWithSso({ provider, redirect, providerId }) {
-  if (!provider) throw new Error("Missing provider");
```
Comment (stephencattaneo) [inline src/sso.js:43]: the same check is done in `getProviderLink`

File: src/login.js [hunk 2]
```diff
@@ -76,7 +78,6 @@ export async function login({
     case "google":
     case "linkedin":
     case "okta":
-      return signonWithSso({ provider: method, redirect });
```
Comment (stephencattaneo) [inline src/login.js:79]: calling with this signature is the same -- `signonWithSso({ provider: method, redirect, providerId });`

**Review by isanchez-userfront — APPROVED** [review]
Simply awesome

File: ts/index.d.ts [hunk 3]
```diff
@@ -1,5 +1,11 @@
 // init
-export declare function init(tenantId: string, options?: object): Promise<void>;
+interface initOptions {
```
Comment (isanchez-userfront) [inline ts/index.d.ts:2]: nit: I'm a bit used to seeing PascalCase for types/interfaces

**Review by isanchez-userfront — APPROVED** [review]

**Pull request merged**